plugins/netlify/netlify.go

```diff
@@ -0,0 +1,26 @@
+package netlify
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func NetlifyCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Netlify CLI",
+    	Runs:      []string{"netlify"},
+		DocsURL:   sdk.URL("https://netlify.com/docs/cli"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
+			needsauth.NotForExactArgs("config"),
```

Comment:
I am sure I see a `config` subcommand for netlify. Is there a reason why this is listed? Not seeing it as a command line flag either. 🤔